notify-api/.vscode/launch.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
I think its about time we create a .workplace file to list all the projects and have a centralized launch.json, just a suggestion not needed for this PR